about.html

```diff
@@ -0,0 +1,170 @@
+</head>
+
+<body>
+    <header>
+        <div class="container-fluid top-bar d-none d-md-block d-lg-block fixed-top">
+            <div class="container d-flex justify-content-end align-items-center" id="top-bar-box">
+                <ul class="d-flex" id="top-bar-social">
+                    <li class="social-item"><a href="#"><i class="fab fa-facebook-f social-icons"></i></a></li>
+                    <li class="social-item"><a href="#"><i class="fab fa-twitter social-icons"></i></a></li>
+                    <li class="social-item lang"><a href="#">English</a></li>
+                    <li class="social-item lang"><a href="#">My Page</a></li>
+                    <li class="social-item lang"><a href="#">Logout</a></li>
+                </ul>
+            </div>
+        </div>
+        <nav class="navbar navbar-expand-md fixed-top" id="nav-container">
+            <div class="container">
+                <a class="navbar-brand text-dark d-none d-md-block d-lg-block" href="#">
+                    <img src="images/logo.png" alt="logo" id="logo">
+                </a>
+                <button class="navbar-toggler collapsed" type="button" data-bs-toggle="collapse"
+                    data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false"
+                    aria-label="Toggle navigation">
+                    <i class="fas fa-bars" id="nav-toggle"></i>
+                </button>
+                <div class="navbar-collapse collapse" id="navbarCollapse">
+                    <ul class="navbar-nav ms-lg-auto ms-md-auto nav-space">
+                        <li class="nav-item">
+                            <a class="nav-link"  href="index.html">Home</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link active" aria-current="page" href="about.html">About</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Products</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">News</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="#">Contact Us</a>
+                        </li>
+                    </ul>
+                    <div class="d-flex justify-content-center">
+                        <button type="button" class="btn btn-outline-primary nav-btn1"> Get In Touch</button>
+                    </div>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="container-fluid main-bg">
+            <div class="container d-flex flex-column align-items-center main-heading">
+                <h1 class="pt-5">"Hello! Sharing World"</h1>
+                <h1 class="display-1 text-center">Aanchal Global<br> Impex Expo 2021</h1>
```

Comment:
- [ ] As requested about the font-weight in the index HTML file kindly change it also here to match the design.